x-pack/test/api_integration/deployment_agnostic/configs/serverless/oblt.logs_essentials.serverless.config.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { createServerlessTestConfig } from '../../default_configs/serverless.config.base';
+
+export default createServerlessTestConfig({
+  serverlessProject: 'oblt',
+  tier: 'oblt_logs_essentials',
+  testFiles: [require.resolve('./oblt.logs_essentials.index.ts')],
+  junit: {
+    reportName: 'Serverless Observability Essentials - Deployment-agnostic API Integration Tests',
```

Comment:
Are we missing `Logs` in the report name?

```suggestion
    reportName: 'Serverless Observability Logs Essentials - Deployment-agnostic API Integration Tests',
```

Reply:
Good catch. Initially, it was only `essentials` everywhere, but with the renaming of the tier name to `logs_essentials`, I've also updated all the test file names and report names. This one fell through the cracks. Updated in [b77b038](https://github.com/elastic/kibana/pull/223612/commits/b77b038f4e8755a0579dba4dc18c7ff9e6cdb006).